tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.DirectLogSubmission.cs

```diff
@@ -79,6 +79,13 @@ internal static class DirectLogSubmission
             /// </summary>
             /// <seealso cref="DirectLogSubmissionSettings.BatchPeriod"/>
             public const string BatchPeriodSeconds = "DD_LOGS_DIRECT_SUBMISSION_BATCH_PERIOD_SECONDS";
+
+            /// <summary>
+            /// Configuration key to disable direct log submission for Azure Functions host.
+            /// Default value is <c>false</c>.
+            /// </summary>
+            /// <seealso cref="DirectLogSubmissionSettings.DisableForAzureFunctionsHost"/>
+            public const string DisableForAzureFunctionsHost = "DD_LOGS_DIRECT_SUBMISSION_DISABLE_FOR_AZURE_FUNCTIONS_HOST";
```

Comment:
Our config naming convention is usually `DD_<SOMETHING>_ENABLED`. Consider calling this
```
DD_LOGS_DIRECT_SUBMISSION_AZURE_FUNCTIONS_HOST_ENABLED
```

If you change the name, remember to flip the default value, too.